### PR TITLE
Some more PowerBI pydantic and import fixes

### PR DESCRIPTION
**Pull request by eavanvalkenburg**

### CODE REVIEW

File: langchain/utilities/powerbi.py [hunk 1]
```diff
@@ -16,6 +16,15 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+try:
```
Comment (dev2049) [inline langchain/utilities/powerbi.py:19]: not super familiar with python import execution, will this only be run if someone tries importing from langchain.utilities.powerbi or would it also be run if someone tried importing something unrelated from langchain.utilities (which imports from langchain.utilities.powerbi)

File: langchain/utilities/powerbi.py [hunk 2]
```diff
@@ -16,10 +16,6 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-if TYPE_CHECKING:
```
Comment (dev2049) [inline langchain/utilities/powerbi.py:19]: why remove this?

Comment (dev2049) [reply langchain/utilities/powerbi.py:19]: (this only affects linting, won't cause any runtime issues)

File: langchain/tools/powerbi/tool.py [hunk 3]
```diff
@@ -147,14 +147,22 @@ class InputToQueryTool(BaseTool):
     """  # noqa: E501
     llm_chain: LLMChain
     powerbi: PowerBIDataset = Field(exclude=True)
-    template: str = QUESTION_TO_QUERY
-    examples: str = DEFAULT_FEWSHOT_EXAMPLES
+    template: Optional[str]
```
Comment (hwchase17) [inline langchain/tools/powerbi/tool.py:150]: why change?

File: langchain/utilities/powerbi.py (hunk 2 above)
Comment (hwchase17) [reply langchain/utilities/powerbi.py:19]: +1

Comment (eavanvalkenburg) [reply langchain/utilities/powerbi.py:19]: fixed by using a different typing class from azure.core, still needs to be regular import in order to get through validations at runtime.

**Review by hwchase17 — APPROVED** [review]
thanks!

**Pull request merged**